config/gulp/website.js

```diff
@@ -10,91 +12,122 @@ var taskBuild = task + ':build';
 var taskServe = task + ':serve';
 var taskDev = task + ':development';
 
-// Remove the .bundle/config file generated by `local-gem`
-//
-gulp.task('local-bundle-clean', function (done) {
+gulp.task('clean-generated-assets', function (done) {
+  return del([
+    'docs/_scss',
```

Comment:
They are copied over in a new `copy-assets` task which has some moving and copying going on.
https://github.com/18F/web-design-standards/blob/rm-jekyll-gem/config/gulp/website.js#L15-L126